client/components/credit-card/stored-card.jsx

```diff
@@ -0,0 +1,45 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+class StoredCard extends React.Component {
+	static propTypes = {
+		card: PropTypes.string.isRequired,
+		card_type: PropTypes.string.isRequired,
+		name: PropTypes.string.isRequired,
+		expiry: PropTypes.string.isRequired,
+	};
+
+	render() {
+		const { card, card_type, name, expiry } = this.props;
+		const expirationDate = this.props.moment( expiry ).format( 'MM/YY' );
+		const cardClasses = classNames(
+			'credit-card__stored-card',
+			card_type && card_type.toLowerCase()
```

Comment:
I left a comment about this elsewhere but maybe it go buried. being explicit here helps because in this construction we have no expectation on what values should appear here.

Reply:
👍 Done in https://github.com/Automattic/wp-calypso/pull/24485/commits/646bea9c3b0278b73c80ce0cf34253645391eb02